### PR TITLE
Add Views

### PR DESCRIPTION
**Pull request by alicemirigo92**
- [ ] Use preprocessed HTML file with embedded Ruby code.
- [ ] Use layouts and templates for shared content.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,22 @@
...
+<h1>A given User detail with posts</h1>
+<%= render 'shared/user', user:@user%>
+
+<div class="user_bio">
+ <h3>Bio</h3>
+ <p><%= @user.bio %><p>
+</div>
+
+<div class="user_post">
+   <% @user.most_three_recent_post.each do |post| %>
```
Comment (elisha2kyakpo1) [inline app/views/users/show.html.erb]: - [ ] Kindly fix the typo for the user model method, this method name should be the same as the one in the user model for the app to work.

method-error

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,18 @@
-<h1>list of posts for a given user </h1>
\ No newline at end of file
+<h1>list of posts for a given user </h1>
+<%= render 'shared/user', user:@user%>
+ <div class="post_list">
+   <% @posts.each do |post| %>
```
Comment (elisha2kyakpo1) [inline app/views/posts/index.html.erb:4]: - [ ] I got an error when I click the button "See all posts", kindly fix the error and submit for a review.

posts

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @alicemirigo92, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive commit messages.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Adding commands for creating and running database migrations.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (Whoistolu) [reply app/views/posts/index.html.erb:4]: - [ ] [REQUIRED] Good job @alicemirigo92 fixing the **See all Post** button and making sure it opens the user's post index, however, you need to fix the UI of the user's post index page to look like the wireframe provided.

### Your Design
Screenshot 2023-05-18 at 21 18 00
### Wireframe Provided
Screenshot 2023-05-18 at 21 17 30

File: app/views/users/show.html.erb (hunk 1 above)
Comment (Whoistolu) [reply app/views/users/show.html.erb]: - [ ] Well resolved 🥇 
https://github.com/alicemirigo92/blog_post/blob/4d144e60418f6c59294f9a70796a4f2b543aea1d/app/views/users/show.html.erb#L10

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @alicemirigo92 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

- [ ] [OPTIONAL] Kindly consider adding a little more description to your PR. What you have is fine but can be better if you add more description to it.

Cheers, and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @alicemirigo92 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**